test/helm_basic_example_template_test.go

```diff
@@ -35,8 +39,11 @@ func TestHelmBasicExampleTemplateRenderedDeployment(t *testing.T) {
 	helmChartPath, err := filepath.Abs("../examples/helm-basic-example")
 	require.NoError(t, err)
 
-	// Since we aren't deploying any resources, there is no need to setup kubectl authentication, helm home, or
-	// namespaces
+	// Since we aren't deploying any resources, there is no need to setup kubectl authentication or helm home.
+
+	// Set up the namespace; confirm that the template renders the expected value for the namespace.
+	namespaceName := "medieval-" + strings.ToLower(random.UniqueId())
+	fmt.Printf("Namespace: %s\n", namespaceName)
```

Comment:
Debug statements should be using `logger.Logf`. Otherwise, it won't be captured by the `terratest_log_parser`.

Reply:
Thanks @yorinasub17! I've corrected this in https://github.com/gruntwork-io/terratest/pull/397/commits/e55d8ce7741626d58db1bf8f2407fe11ab180707. Mind taking another look?